gc/base/VirtualMemory.hpp

```diff
@@ -177,6 +177,18 @@ class MM_VirtualMemory : public MM_BaseVirtual {
 		_consumerCount -= 1;
 	}
 
+#if defined(OMR_GC_DOUBLE_MAP_ARRAYLETS)
+	/**
+	 * Sets double map API available to true if memfd_create is available in the system
+	 */
+	MMINLINE void updateDoubleMapping()
+	{
+		if (0 != (OMRPORT_VMEM_MEMORY_MODE_DOUBLE_MAP_AVAILABLE & _identifier.mode)) {
+			_extensions->isDoubleMapAPIAvailable = true;
+		}
+	}
+#endif /* defined(OMR_GC_DOUBLE_MAP_ARRAYLETS) */
```

Comment:
I would prefer NOT to do settings like this in `MM_VirtualMemory`. Instead please make this function `bool` to check flag only and do setting at caller side. Also with this you don't need `#if defined(OMR_GC_DOUBLE_MAP_ARRAYLETS)`